tests/integration/src/test/java/org/mule/util/VariableValueTestCase.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.util;
+
+import static org.hamcrest.CoreMatchers.is;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.junit.Assert.assertThat;
+
+import org.mule.api.MuleMessage;
+import org.mule.api.client.LocalMuleClient;
+import org.mule.tck.junit4.FunctionalTestCase;
+import org.mule.tck.junit4.rule.SystemProperty;
+
+
+import org.junit.Rule;
+import org.junit.Test;
+
+
+public class VariableValueTestCase extends FunctionalTestCase
```

Comment:
Can't you test this with a unti test? (i.e: a class extending from AbstractMuleTestCase instead of FunctionalTestCase)